source/api.rst

```diff
@@ -26,6 +26,25 @@ You can refer to the next documentation to get the list of available endpoints.
 
 .. _websockets:
 
+Errors
+======
+
+Symbol uses conventional HTTP response codes to indicate the success or failure of an API request.
+
+Codes in the ``2xx`` range indicate success.
+Codes in the ``4xx`` range indicate an error occurred with the information provided by the user.
+Codes in the ``5xx`` range indicate an error with Symbol's server.
+
+.. csv-table:: HTTP Status Code Summary
+    :header: "Error Code", "Description"
+    :delim: ;
+
+    200/202 - Success; Everything worked as expected
```

Comment:
Split 200 and 202: 

200; OK; ...
202; Accepted; The request has been accepted, but the processing has not been completed.